src/unquote/index.js

```diff
@@ -0,0 +1,16 @@
+// copied from https://github.com/lakenen/node-unquote
+
+var reg = /['"]/;
```

Comment:
I suspect this regexp will replace only the quotes and leave the backslashes. Shouldn't it be `[\'\"]`?

Reply:
eslint tells me it can be removed.

Reply:
![image](https://user-images.githubusercontent.com/8327041/126773619-2c72ca7d-87e0-411b-8c8a-bb49c146750f.png)

Reply:
My bad, I have read the code not thorough enough the first time. Everything is ok

Reply:
@madyankin If you want to keep it like where copied from, I will change it to `[\'\"]` and add a `// eslint-disable no-useless-escape` line to disable this rule.

Reply:
@madpilot ok.